lib/cisco_node_utils/cmd_ref/vdc.yaml

```diff
@@ -7,8 +7,8 @@ _exclude: [N3k, N5k, N6k, N9k]
 
 _template:
   get_command: 'show run vdc all'
-  get_context: ['/^vdc <vdc>/']
-  set_context: ['vdc <vdc>']
+  get_context: ['(?)/^vdc <vdc>/']
+  set_context: ['terminal dont-ask ; vdc <vdc>']
```

Comment:
Does this have to be spliced with ';' to work properly, or can this be a normal multi-entry context `set_context: ['terminal dont-ask', 'vdc <vdc>']`?